twilight-model/src/gateway/payload/incoming/typing_start.rs

```diff
@@ -123,14 +123,6 @@ impl<'de> Visitor<'de> for TypingStartVisitor {
             %user_id,
         );
 
-        let member = if let (Some(guild_id), Some(member)) = (guild_id, member) {
-            tracing::trace!(%guild_id, ?member, "setting member guild id");
-
-            Some(member.into_member(guild_id))
-        } else {
-            None
-        };
-
```

Comment:
This whole manual implementation can now be replaced with `#[derive(Deserialize)]` (part of #1364)